Adding section on upgrading Istio in an air-gapped environment (#285)
rancher/rancher-docs: docs/pages-for-subheaders/istio.md

## Additional Steps for Installing Istio on an RKE2 Cluster

To install Istio on an RKE2 cluster, follow the steps in [this section.](../integrations-in-rancher/istio/configuration-options/install-istio-on-rke2-cluster.md)

## Upgrading Istio in an Air-Gapped Environment

In Rancher v2.6.x and up, Istio may be upgraded in an air-gapped environment.

The Istio pod security policy is now enabled by default, and a new value, `installer.releaseMirror.enabled`, has been added to the rancher-istio chart to enable and disable the server that supports air-gapped upgrades. Note that `installer.releaseMirror.enabled` is set to `false` by default and must be changed as needed during install or upgrade. Follow the steps below:

1. Provision an air-gapped Rancher 2.6.x instance and an air-gapped custom cluster in the Rancher UI.
2. Install Monitoring in the cluster: **Cluster Explorer -> Apps & Marketplace -> Charts -> Monitoring**.

> 3. Pull all required images for Istio in the private registry setup that will be used in the air-gapped environment.

Reviewer: ```suggestion
3. Pull all required images for Istio into the private registry you will use in the air-gapped environment.
```

4. Install Istio in the cluster: **Cluster Explorer -> Apps & Marketplace -> Charts -> Istio**.

:::note

On the fresh Istio install, [Jaeger](https://www.jaegertracing.io/[) and [Kiali](https://kiali.io/) may be enabled if desired. To ensure Jaeger and Kiali work, ensure that `installer.releaseMirror.enabled=true` is set in the values.yaml at install.

:::

5. Upgrade the Istio installation.

:::caution

> If not done by this point, you must set the value `installer.releaseMirror.enabled=true` to allow for the upgrade.

Reviewer: ```suggestion
If you haven't already, set `installer.releaseMirror.enabled=true` to upgrade Istio.
```
